.github/workflows/IntegrationTest.yml

```
@@ -0,0 +1,21 @@
+name: "IntegrationTest"
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags: '*'
+  pull_request:
+
+jobs:
+  integration-test:
+    name: "IntegrationTest"
+    strategy:
+       matrix:
+         repo:
+           - 'ITensor/BlockSparseArrays.jl'
+           - 'ITensor/SymmetrySectors.jl'
+    uses: "ITensor/ITensorActions/.github/workflows/IntegrationTest.yml@main"
+    with:
+      local-registry-urls: "https://github.com/ITensor/ITensorRegistry.git"
```

```suggestion
      localregistry: "https://github.com/ITensor/ITensorRegistry.git"
```
I updated the syntax: https://github.com/ITensor/ITensorActions?tab=readme-ov-file#integrationtest.